README.md

```diff
@@ -24,17 +24,17 @@ This module provides Cloud Trace support for Node.js applications. [Google Cloud
 
         npm install --save @google/cloud-trace
 
-2. Include and start the library at the *top of the main script of your application*. It's important that Cloud Trace is the first thing executed so that it can accurately gather data:
+2. If you are running locally, or not on the Google Cloud Platform, set the GCLOUD_PROJECT environment variable. Your project ID is visible in the [Google Cloud Console Console](https://console.cloud.google.com/project?_ga=1.258049870.576536942.1443543237), it may be something like `particular-future-12345`. If your application is [running on Google Cloud Platform](running-on-google-cloud-platform), you don't need to specify the project ID.
```

Comment:
In the quick start instructions, for 2, I would just ask them to give us the project id - no hemming and hawing :)  Something like:

Set the GCLOUD_PROJECT environment variable. You can find your Project ID in the Google Cloud Developers Console, or by running the command `gcloud projects list`.  You can ensure this environment variable is set at startup time by placing it in your startup script in `package.json`:

```json
"scripts": {
  "start": "GCLOUD_PROJECT=<YOUR_PROJECT_ID> node server.js",
},
```

Then, further down in the readme, include a specific section (if there isn't one already) on running this outside of Google Cloud.